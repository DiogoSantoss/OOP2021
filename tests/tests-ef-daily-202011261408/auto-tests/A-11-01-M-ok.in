7
4
P1
4
P0
0
0
