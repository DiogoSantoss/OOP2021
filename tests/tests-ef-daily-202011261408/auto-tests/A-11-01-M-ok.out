Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Fornecedores
1 - Mostrar fornecedores
2 - Registar
3 - Permitir/inibir transacções
4 - Consultar transacções
0 - Sair
Escolha uma opção: Identificador do fornecedor: Gestão de Fornecedores
1 - Mostrar fornecedores
2 - Registar
3 - Permitir/inibir transacções
4 - Consultar transacções
0 - Sair
Escolha uma opção: Identificador do fornecedor: Consultar transacções: Operação inválida: O fornecedor 'P0' não existe.
Gestão de Fornecedores
1 - Mostrar fornecedores
2 - Registar
3 - Permitir/inibir transacções
4 - Consultar transacções
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 